blivet/devicefactory.py

```diff
@@ -2095,13 +2103,56 @@ def _get_total_space(self):
 
             return space
```

Comment:
Is there a reason that this branch is the only one returning a result?

Reply:
Yes, this `return` is unnecessary, the `return` at the end of the function will take care of all branches. Fixed, thanks.